MoviesExample/app/src/main/java/com/novoda/demo/movies/MoviesAdapter.java

```diff
@@ -81,44 +79,37 @@ public long getItemId(final int position) {
 
     class MovieItem extends RecyclerView.ViewHolder {
 
-        @BindView(R.id.movie_item_title) TextView text;
-        @BindView(R.id.movie_item_poster) ImageView poster;
-        @BindView(R.id.movie_item_rating) TextView rating;
-
-        MovieItem(View itemView, final Listener listener) {
-            super(itemView);
-            ButterKnife.bind(this, itemView);
-            itemView.setOnClickListener(new View.OnClickListener() {
-                @Override
-                public void onClick(final View v) {
-                    int adapterPosition = getAdapterPosition();
-                    if (adapterPosition != -1) {
-                        listener.onMovieSelected(moviesSate.get(adapterPosition));
-                    }
-                }
-            });
+        MoviesListItemBinding binding;
+
+        MovieItem(MoviesListItemBinding binding, final Listener listener) {
+            super(binding.getRoot());
+            this.binding = binding;
+            binding.setListener(listener);
```

Comment:
Before this, there was a click listener to capture the click and then make a call to `listener.onMovieSelected`: https://github.com/novoda/android-demos/pull/42/files#diff-09834a583d469601e211f59084c37f3eL91


With Data Binding, we can call `listener.onMovieSelected` directly: https://github.com/novoda/android-demos/pull/42/files#diff-7a051a014987e81424304f4f8bd9e2afR20